packages/fxa-content-server/tests/functional/oauth_sign_up.js

```diff
@@ -13,33 +13,25 @@ const config = intern._config;
 var PASSWORD = 'password12345678';
 const SUCCESS_URL = config.fxaContentRoot + 'oauth/success/dcdb5ae7add825d2';
 var email;
-var bouncedEmail;
+//var bouncedEmail;
 
 const {
-  click,
   clearBrowserState,
-  closeCurrentWindow,
   fillOutEmailFirstSignUp,
-  getFxaClient,
+  fillOutSignUpCode,
+  //  getFxaClient,
   noEmailExpected,
-  noSuchElement,
-  openExternalSite,
   openFxaFromRp,
   openPage,
-  openVerificationLinkInDifferentBrowser,
-  openVerificationLinkInNewTab,
-  openVerificationLinkInSameTab,
-  switchToWindow,
   testElementExists,
-  testElementTextInclude,
   testUrlInclude,
-  visibleByQSA,
+  //  visibleByQSA,
```

Comment:
Is this fixed in the separate PR to add email bouncing?

Reply:
Ahha, yes, I can bring this back now!